### PR TITLE
Patron Vesting deployment

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by zeus-infinex** [comment]
## [testnets](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/4600eaf/report/testnets/) <span title="passed">✅214</span> | <span title="failed">❌12</span> | <span title="skipped">⏩6</span>
[Markdown](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/4600eaf/report/testnets/report.md) | [JSON](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/4600eaf/report/testnets/report.json)
<details>
<summary>gp</summary>

### GovernancePoints (base testnets) GovernancePoints has authorised transfers only enabled
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/gp.test.ts#L84

```
AssertionError: expected false to be true // Object.is equality
```
</details>
<details>
<summary>multichain</summary>

### Multichain (blast testnets) Initial Deployment Infinex Protocol Config Beacon points to valid USDC contract
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/multichain.test.ts#L183

```
AssertionError: expected '0x42000000000000000000000000000000000…' to be '0x43000000000000000000000000000000000…' // Object.is equality
```
</details>
<details>
<summary>patron-app</summary>

### Patron Purchase (arbitrum testnets) Check Ownership Transferred Successfully App Beacon
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/patron-app.test.ts#L120

```
AssertionError: expected '0x6298551D56F825B3b6c3350D32129Ffd3cF…' to be undefined // Object.is equality
```

### Patron Purchase (arbitrum testnets) Check Ownership Transferred Successfully Patron Point of Purchase
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/patron-app.test.ts#L126

```
AssertionError: expected '0x6298551D56F825B3b6c3350D32129Ffd3cF…' to be undefined // Object.is equality
```

### Patron Purchase (base testnets) Check Ownership Transferred Successfully App Beacon
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/patron-app.test.ts#L120

```
AssertionError: expected '0x6298551D56F825B3b6c3350D32129Ffd3cF…' to be undefined // Object.is equality
```

### Patron Purchase (base testnets) Check Ownership Transferred Successfully Patron Point of Purchase
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/patron-app.test.ts#L126

```
AssertionError: expected '0x6298551D56F825B3b6c3350D32129Ffd3cF…' to be undefined // Object.is equality
```

### Patron Purchase (ethereum testnets) Check Ownership Transferred Successfully App Beacon
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/patron-app.test.ts#L120

```
AssertionError: expected '0x6298551D56F825B3b6c3350D32129Ffd3cF…' to be undefined // Object.is equality
```

### Patron Purchase (ethereum testnets) Check Ownership Transferred Successfully Patron Point of Purchase
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/patron-app.test.ts#L126

```
AssertionError: expected '0x6298551D56F825B3b6c3350D32129Ffd3cF…' to be undefined // Object.is equality
```

### Patron Purchase (optimism testnets) Check Ownership Transferred Successfully App Beacon
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/patron-app.test.ts#L120

```
AssertionError: expected '0x6298551D56F825B3b6c3350D32129Ffd3cF…' to be undefined // Object.is equality
```

### Patron Purchase (optimism testnets) Check Ownership Transferred Successfully Patron Point of Purchase
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/patron-app.test.ts#L126

```
AssertionError: expected '0x6298551D56F825B3b6c3350D32129Ffd3cF…' to be undefined // Object.is equality
```

### Patron Purchase (polygon testnets) Check Ownership Transferred Successfully App Beacon
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/patron-app.test.ts#L120

```
AssertionError: expected '0x6298551D56F825B3b6c3350D32129Ffd3cF…' to be undefined // Object.is equality
```

### Patron Purchase (polygon testnets) Check Ownership Transferred Successfully Patron Point of Purchase
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/patron-app.test.ts#L126

```
AssertionError: expected '0x6298551D56F825B3b6c3350D32129Ffd3cF…' to be undefined // Object.is equality
```
</details>


## [staging](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/4600eaf/report/staging/) <span title="passed">✅229</span> | <span title="failed">❌2</span> | <span title="skipped">⏩6</span>
[Markdown](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/4600eaf/report/staging/report.md) | [JSON](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/4600eaf/report/staging/report.json)
<details>
<summary>gp</summary>

### GovernancePoints (base staging) GovernancePoints has authorised transfers only enabled
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/gp.test.ts#L84

```
AssertionError: expected false to be true // Object.is equality
```
</details>
<details>
<summary>multichain</summary>

### Multichain (blast staging) Initial Deployment Infinex Protocol Config Beacon points to expected app registry contract
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/multichain.test.ts#L137

```
ContractFunctionExecutionError: HTTP request failed.
```
</details>


## [mainnets](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/4600eaf/report/mainnets/) <span title="passed">✅224</span> | <span title="failed">❌7</span> | <span title="skipped">⏩6</span>
[Markdown](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/4600eaf/report/mainnets/report.md) | [JSON](https://sys.infinex.xyz/reports/evm-contracts/deployments-test/4600eaf/report/mainnets/report.json)
<details>
<summary>gp</summary>

### GovernancePoints (base mainnets) GovernancePoints has authorised transfers only enabled
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/gp.test.ts#L84

```
AssertionError: expected false to be true // Object.is equality
```

### GovernancePoints (base mainnets) GovernancePoints has the GPDistributor added as an authorised transferrer
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/gp.test.ts#L98

```
AssertionError: expected false to be true // Object.is equality
```

### GovernancePoints (base mainnets) GPDistributor has the correct owner
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/gp.test.ts#L116

```
ContractFunctionExecutionError: The contract function "owner" returned no data ("0x").
```

### GovernancePoints (base mainnets) GPDistributor has the correct GPToken address
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/gp.test.ts#L122

```
ContractFunctionExecutionError: The contract function "GPToken" returned no data ("0x").
```

### GovernancePoints (base mainnets) GPDistributor has the correct authorised keeper
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/gp.test.ts#L126

```
ContractFunctionExecutionError: The contract function "authorizedKeepers" returned no data ("0x").
```
</details>
<details>
<summary>multichain</summary>

### Multichain (blast mainnets) Council executed steps completed Ownership Infinex Protocol Config Beacon owner matches config
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/multichain.test.ts#L343

```
ContractFunctionExecutionError: HTTP request failed.
```

### Multichain (blast mainnets) Council executed steps completed Ownership App registry has no pending owner
https://github.com/infinex-xyz/infinex-deployments/blob/main/workspace/test/tests/multichain.test.ts#L358

```
ContractFunctionExecutionError: HTTP request failed.
```
</details>

File: infinex-patron/vesting/infinex-patron-vesting.ethereum.toml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+[invoke.addTier5]
+target = ["InfinexPatronVesting.PatronVesting"]
+func = "addTier"
+args = [5, "0xbE62686bDaE4b938e40e5A09C4e2Be3B5Bc74ba9", 0, 31536000, 63072000, 1]
+fromCall.func = "owner"
+
+[invoke.addTier6]
+target = ["InfinexPatronVesting.PatronVesting"]
+func = "addTier"
+args = [6, "", 0, 31536000, 63072000, 2]
```
Comment (ramboinfinex) [inline infinex-patron/vesting/infinex-patron-vesting.ethereum.toml]: is "" address(0)?

File: infinex-patron/vesting/1-patron-vesting-configuration.ethereum.toml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+version = "1.0.1"
+description = "Configuration for the Infinex Patron Vesting"
+
+[pull.InfinexPatronDistributor]
+source = "infinex-patron-vesting:1.0.1@patron-vesting"
+
+[invoke.SetTier1StartTime]
+target = ["<%= imports.InfinexPatronVesting.imports.InfinexPatronVesting.contracts.PatronVesting.address %>"]
+func = "setTierStartTime"
+args = [1, 0] #
```
Comment (lemoninfinex) [inline infinex-patron/vesting/1-patron-vesting-configuration.ethereum.toml:14]: why are we doing this? also there's a `#` every where which looks like there was a comment before but got remove

Comment (rayInfinex) [reply infinex-patron/vesting/1-patron-vesting-configuration.ethereum.toml:14]: I think there was a comment before and then it got removed

**Review by ramboinfinex — APPROVED** [review]

**Pull request merged**